src/mesh/Mesh.cpp

```diff
@@ -481,3 +482,7 @@ std::ostream &operator<<(std::ostream &os, const Mesh &m)
 
 } // namespace mesh
 } // namespace precice
+
+template <>
+struct fmt::formatter<precice::mesh::Mesh> : ostream_formatter {
+};
```

Comment:
Do we actually ever print the _entire_ Mesh?

Reply:
No, I removed it.